src/plugins/plugin-http.ts

```diff
@@ -183,7 +183,8 @@ function patchHttp(http: HttpModule, api: TraceAgent) {
   }
 }
 
-// https.get depends on https.request in <8.9 and >=8.9.1
+// https.get depends on Node http internals in 8.9.0 and 9+ instead of the
```

Comment:
The actual code is `<8.9.0 || ^8.9.1` while this comment says `8.9.0 and 9+`. Is it intentional?

Reply:
`patchHttps` is actually used in the patch for `<8.9.0 || ^8.9.1`. (Not really related, but the comment was inaccurate before.)